donate/core/tests/test_models.py

```diff
@@ -24,6 +24,11 @@ def test_get_initial_currency_uses_header(self):
             'clp'
         )
 
+    def test_serve_sets_subscribed_cookie(self):
+        request = RequestFactory().get('/?subscribed=1')
+        response = DonationPage().serve(request)
+        self.assertEqual(response.cookies['subscribed'].value, '1')
+
```

Comment:
@solarissmoke Since the setting of the cookie is conditional, could we also add a test to show that it is NOT set when 'subscribed' is not present, or has a value other than '1'?

Reply:
Yep, good point. Added.